portfolio/src/main/webapp/style.css

```diff
@@ -1,15 +1,142 @@
+h1 {
```

Comment:
It could be useful to split up your css file into sections (with comments denoting each section) where each section. It makes it easier to follow. The good thing is that your css file is almost organized that to begin with!